Tasks/TwineAuthenticateV1/twineauthenticatemain.ts

```diff
@@ -72,6 +139,26 @@ function findDuplicatesInArray<T>(array: Array<T>): Array<T>{
     return array.filter((e, i, a) => a.indexOf(e) !== i);
 }
 
+function removeFromFeedCount(internalFeed: auth.AuthInfo[], externalEndpoints: auth.AuthInfo[], entry: auth.AuthInfo): void {
+    if (internalFeed.includes(entry)) {
+        internalFeed.pop();
+        return;
+    }
+    externalEndpoints.pop();
+}
+
+function getNestedRepoProperty(connection: object): string {
+    for (const key in  connection) {
+        if (typeof connection[key] === 'object') {
+            return getNestedRepoProperty(connection[key]);
+        }
+        else if (key === 'repository') {
+            return connection[key];
+        }
+    }
+    return tl.loc("NoRepoFound");
```

Comment:
should this return a boolean instead ?

Reply:
This function is supposed to return a repo url as a string, so if a section doesn't have a repo then I'm returning a default string

Reply:
I would have the method return string | undefined, and then return undefined instead of a tl loc.

Reply:
Ok, changed this!

Reply:
Hmm I'm not seeing the suggested changes wrt returning undefined

Reply:
Thanks for noticing! I made updates and it's there now!